website/docs-scraper/extract-content-from-markdown.mjs

```diff
@@ -0,0 +1,217 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// remark
+import { remark } from 'remark';
+import { visit } from 'unist-util-visit';
+import { selectAll } from 'unist-util-select';
+import { fromHtml } from 'hast-util-from-html';
+import { fromMarkdown } from 'mdast-util-from-markdown';
+import { toString } from 'mdast-util-to-string';
+
+import handlebars from 'handlebars';
+
+import _ from 'lodash';
+
+// plugins
+import remarkGfm from 'remark-gfm';
+import remarkHtml from 'remark-html';
+
+// local/custom
+import { WCAG_CRITERIA } from './parts/getWcagCriteria.mjs';
+import { replaceDocTags } from './parts/replaceDocTags.mjs';
+import { remarkRemoveComments } from './parts/remarkRemoveComments.mjs';
+import { remarkRemoveCodeBlocks } from './parts/remarkRemoveCodeBlocks.mjs';
+import { remarkProcessCustomImageFormat } from './parts/remarkProcessCustomImageFormat.mjs';
+import { remarkStripHeliosContentBlocksDelimiters } from './parts/remarkStripHeliosContentBlocksDelimiters.mjs';
+import { remarkStripHeliosHandlebarsExpressions } from './parts/remarkStripHeliosHandlebarsExpressions.mjs';
+// import { remarkStripHeliosReleaseNotesMetadata } from './remarkStripHeliosReleaseNotesMetadata.mjs';
```

Comment:
does this mean we aren't currently indexing the release notes page? if not, do we have a task in the backlog to track that so we can figure that out?

Reply:
No, we do index them, they're part of the processed markdown. This mini-plugin is to remove the metadata from them (PR number and thanks to external contributors)